pkg/config/cos.go

```diff
@@ -444,6 +429,90 @@ func updateBond(stage *yipSchema.Stage, name string, network *Network) error {
 	return nil
 }
 
+func updateBridge(stage *yipSchema.Stage, name string, mgmtNetwork *Network) error {
+	// add Bridge named MgmtInterfaceName and attach Bond named MgmtBondInterfaceName to bridge
+
+	needVlanInterface := false
+	// pvid is always 1, if vlan id is 1, it means untagged vlan.
+	if mgmtNetwork.VlanID >= 2 && mgmtNetwork.VlanID <= 4094 {
```

Comment:
it's better to define a const for 4094, it's scattered in the code

Reply:
I will not change this in this stage

Reply:
There are some ranges between 0 to 4094 and 2 to 4094.
It will confuse if we only set a variable for 4094 but not for 0 and 2.
and it's hard to give a name for 0 and 2.